ci-operator/step-registry/acm/interop-p2p/submariner-addon/acm-interop-p2p-submariner-addon-commands.sh

```
@@ -0,0 +1,507 @@
+#
+set -euxo pipefail; shopt -s inherit_errexit
+
+eval "$(
+    typeset -a _fURL=()
+    type -t wget 1>/dev/null && _fURL=(wget -nv -O-) || _fURL=(curl -fsSL)
+    "${_fURL[@]}" https://raw.githubusercontent.com/RedHatQE/OpenShift-LP-QE--Tools/f63f1f606b1d76f6ef2a3e78b4ec1ad7362d4fac/libs/bash/common/EnsureReqs.sh
+)"; EnsureReqs jq
+
+if [[ -n "${SHARED_DIR}" && -s "${SHARED_DIR}/proxy-conf.sh" ]]; then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)